libs/math/src/math/linalg/blas/gemm_tn_vector.cpp

```diff
@@ -106,6 +106,14 @@ template class Blas<float, Signature(_C <= _alpha, _A, _B, _beta, _C),
                     Computes(_C <= _alpha * T(_A) * _B + _beta * _C),
                     platform::Parallelisation::VECTORISE>;
 
+template class Blas<
+    fetch::fixed_point::FixedPoint<16, 16>, Signature(_C <= _alpha, _A, _B, _beta, _C),
+    Computes(_C <= _alpha * T(_A) * _B + _beta * _C), platform::Parallelisation::VECTORISE>;
+
+template class Blas<
```

Comment:
missing the fp128 implementation here?

Reply:
fp128 does not -yet- support vectorisation, so it's intentionally omitted.